frappe/integrations/doctype/google_calendar/google_calendar.py

```diff
@@ -683,55 +612,79 @@ def get_recurrence_parameters(recurrence):
 	return frequency, until, byday
 
 
-"""API Response
```

Comment:
Maintaining something like this would've been useful. Unless it wasn't useful while building this :trollface: